docs/version0.37.md

```diff
@@ -33,11 +33,9 @@ The following new features and notable changes since version 0.36.x are included
 
 ### Binaries and supported environments
 
-Eclipse OpenJ9&trade; release 0.36.0 supports OpenJDK 8 and 17.
+Eclipse OpenJ9&trade; release 0.37.0 supports OpenJDK 19.
 
-Release 0.36.1 supports OpenJDK 11.
-
-Support for running OpenJDK 8 and OpenJDK 11 on CentOS 6.10 is deprecated and might be removed in a future release. OpenJ9 will not be tested with OpenJDK 11 on CentOS 6.10 after the 0.36.1 release.
+RHEL 8.4 is now supported but RHEL 8.2 is out of support.
```

Comment:
RHEL 8.4 was always supported, we only list the minimum levels. 8.4 is the new minimum level.